### PR TITLE
New LIT for Zəmmāre za-Hosāʿnā and Zəmmāre za-Dabra Zayt

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created two new ID for Zəmmāre za-Hosāʿnā and Zəmmāre za-Dabra Zayt according to our discussion in Issue #2380. I added a relation to point to the general record of LIT2610Zemmar as it was discussed to treat them as if they were part-texts of the general record (as discussed in Issue #2381).

### CODE REVIEW

File: new/LIT6848ZemmareDabraZa.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6848ZemmareDabraZa" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ዝማሬ፡ ዘደብረ፡ ዘይት፡</title>
+                <title corresp="#t1" xml:lang="gez" type="normalized">Zəmmāre za-Dabra Zayt</title>
+                <title corresp="#t1" xml:lang="en">Chant to the Mount of Olives</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT6848ZemmareDabraZa.xml:10]: I would translate it "Chant for Dabra Zayt". Because the idea is "Chant for (the feast of) Dabra Zayt"

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/LIT6848ZemmareDabraZa.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply new/LIT6848ZemmareDabraZa.xml:10]: Yes, I agree.

**Comment by karljonaskarlsson** [comment]
Again, we are not talking about individual "chants" but about "commemorations" or "sets of chants", right?

**Comment by CarstenHoffmannMarburg** [comment]
@karljonaskarlsson, do you suggest to change the title to "Chants for Dabra Zayt" or "Commemoration for Dabra Zayt" and "Chants of Hosanna" and "Commemoration of Hosanna" respectively?

**Comment by karljonaskarlsson** [comment]
Thank you, Denis, for the correction! I see now in the other issue that we are indeed talking about individual chants.

**Comment by CarstenHoffmannMarburg** [comment]
So, I can keep it like this:
```
<titleStmt>
                <title xml:id="t1" xml:lang="gez">ዝማሬ፡ ዘሆሳዕና፡</title>
                <title corresp="#t1" xml:lang="gez" type="normalized">Zəmmāre za-Hosāʿnā</title>
                <title corresp="#t1" xml:lang="en">Chant of Hosanna</title>
                <title xml:id="t2">Hymn for Palm Sunday</title>
                <editor role="generalEditor" key="AB"/>
                <editor key="CH"/>
                <funder>Akademie der Wissenschaften in Hamburg</funder>
            </titleStmt>
```
```
<titleStmt>
                <title xml:lang="gez" xml:id="t1">ዝማሬ፡ ዘደብረ፡ ዘይት፡</title>
                <title corresp="#t1" xml:lang="gez" type="normalized">Zəmmāre za-Dabra Zayt</title>
                <title corresp="#t1" xml:lang="en">Chant for Dabra Zayt</title>
                <editor role="generalEditor" key="AB"/>
                <editor key="CH"/>
                <funder>Akademie der Wissenschaften in Hamburg</funder>
            </titleStmt>
```

**Comment by thea-m** [comment]
@DenisNosnitsin1970 @karljonaskarlsson can this be merged?

**Comment by CarstenHoffmannMarburg** [comment]
The reason, why I preferred "of" rather than "for" is that it is closer to the relative genitive construction with za-.

**Review by thea-m — APPROVED** [review]

**Pull request merged**